project_collaborator/models/project_project.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+
+    collaborator_ids = fields.Many2many(
```

Comment:
Odoo standard have this field name, isn't? Does this module override the standard project collaborator features?
https://github.com/odoo/odoo/blob/059b19c59a9222b676be5246eb94d680cf6bf27b/addons/project/models/project.py#L373

Reply:
Thanks, for review.
I'll fix it